app/tracker/page.tsx

```diff
@@ -65,35 +98,60 @@ export default function Tracker() {
           <p className="font-normal bg-white text-[20px] text-center text-[#173563] p-[10px] rounded-[10px] shadow-2xl">
             <strong>{orderID}</strong>
           </p>
+
+
+          <div className="text-left text-[#173563] mt-2 ml-4 p-[10px] rounded-[10px] w-fit">
+            <p className="text-lg"><strong>Customer:</strong> {customerDetails.name}</p>
+            <p><strong>Payment Status:</strong> {paymentStatus}</p>
+            <p><strong>Weight:</strong> {customerDetails.weight}</p>
+            <p><strong>Price:</strong> {customerDetails.price}</p>
+          </div>
+
         </div>
+        
 
         <div className="text-left overflow-y-auto mt-[30px]">
-          {orderHistory.length > 0 ? (
-            orderHistory.map((order, index) => (
-              <div key={index} className="relative flex">
-                <div className={`bullet ${statusArray.indexOf(order.status) <= statusArray.indexOf(currentStatus) ? "active" : ""}`}></div>
-                {index < orderHistory.length - 1 && <div className="line"></div>}
-                <div>
-                  <p className="font-normal text-[12.5px] text-[#173563]">
-                    <strong>{order.status}</strong> - {order.message}
-                  </p>
-                  <p className="font-normal text-[12.5px] text-[#173563] mb-5">
-                    {order.date} | {order.time}
-                  </p>
-                </div>
-              </div>
-            ))
-          ) : (
-            <p className="text-[#173563]">No order history available.</p>
-          )}
+{orderHistory.length > 0 ? (
+  orderHistory.map((order, index) => (
+    <div key={order.message} className="relative flex justify-end">
+      <div>
+        <p className="font-normal text-[12.5px] text-[#173563] text-right">
+          {order.message} - <strong>{order.status}</strong>
+        </p>
+        <p className="font-normal text-[12.5px] text-[#173563] mb-5 text-right">
+          {order.date} | {order.time}
+        </p>
+      </div>
+      <div className="flex flex-col items-center ml-5">
+        <div className={`bullet ${index <= currentStatus ? 'active' : ""}`}></div>
+        {index < orderHistory.length - 1 && <div className="line"></div>}
+      </div>
+    </div>
+  ))
+) : (
+  <p className="text-[#173563]">No order history available.</p>
+)}
+
```

Comment:
do you have prettier extension installed? kay tabbing seems not right here hehe.

if wala pa na install, pa install lang sa Extensions nga marketplace in vscode.

if na install na, make sure nga siya ang default formatter and format on save is on